packages/design-system/src/components/Dropdown/__snapshots__/Dropdown.test.tsx.snap

```diff
@@ -2,7 +2,8 @@
 
 exports[`Dropdown dropdown matches snapshot 1`] = `
 <div
-  class=""
+  aria-invalid="false"
+  class="ds-c-fieldset"
```

Comment:
This is the result of a copy-paste error on my part here: https://github.com/CMSgov/design-system/pull/2243/files#diff-c132558c7ba0b4e130880148a65be06ed0cb42a950020183fb88bc11a899a351R116

I don't think we should be passing `wrapperIsFieldset: true`, because it's not a fieldset.

Reply:
Ok I actually went back and forth on this one, thought that might be off, it's hard to know what needs to be a fieldset based on other rules in the code.

Reply:
I was just going off of what produces the fewest changes to existing behavior. I didn't ask myself the question of whether it should be a fieldset or not